bylaws/5-decision-making.md

```diff
@@ -7,60 +7,59 @@ weight: 5
 This section describes the decision-making process, voting, and elections for the CF.
 
 - [General Decision Making](#general-decision-making)
-  - [Seeking consensus in meetings](#seeking-consensus-in-meetings)
-  - [Seeking consensus on mailing lists](#seeking-consensus-on-mailing-lists)
+  - [Expectations for Consensus Decision Making](#expectations-for-consensus-decision-making)
+- [Proposal Development and Review Process](#proposal-development-and-review-process)
+  - [Reviews](#reviews)
 - [Elections](#elections)
 
 ## General Decision Making
 
-True to Open Source roots, the CF ensures every member's voice is heard through the Lazy Consensus decision-making model.
+The CF adopts the Lazy Consensus model, as defined by the [Apache Foundation](https://community.apache.org/committers/decisionMaking.html), combined with [Martha's Rules](https://digitalcommons.unl.edu/cgi/viewcontent.cgi?article=1825&context=sociologyfacpub) adapted for our context. This hybrid approach ensures inclusivity and efficiency in decision-making across time zones.
 
-To quote the [Consensus definition][] from the Apache Foundation:
+### Expectations for Consensus Decision Making
 
-> Consensus
->
-> The word “consensus” is a bit ambiguous in English, and so can lead to some misunderstandings of intent when we use it in the context of project decisions. Consensus does not mean that everyone agrees on all details. Rather, it means that the project, as a whole, has arrived a decision, or at least a compromise, that everyone can live with.
->
-> Lazy Consensus
->
-> Lazy consensus is the first, and possibly the most important, consensus-building tool we have. Essentially lazy consensus means that you don’t need to get explicit approval to proceed, but you need to be prepared to listen if someone objects.
+Consensus is about finding a workable compromise:
+- Dissent should be voiced early for constructive discussion.
+- Every member, irrespective of role, is entitled to an equal voice and vote.
 
-When seeking consensus, members should express objections or dissent as early as possible to ensure there is time to discuss and address objections.
+## Proposal Development and Review Process
 
-Every member, regardless of their role or contribution level, has an equal voice and is entitled to one vote in all decisions.
+Proposals begin as GitHub Discussions to encourage collaborative refinement.
 
-[Consensus definition]: https://community.apache.org/committers/decisionMaking.html
+### Reviews
 
-### Seeking consensus in meetings
+For a proposal ready for review:
 
-When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+1. **Review Discussion Creation:**
+   - Transition from the initial discussion to a Review discussion, outlining the proposal in detail.
```

Comment:
Does this transition result in two GitHub Discussions? One for the initial collaboration, and another for the review? With the Review discussion linked back to the collaboration one for reference?

Reply:
Yes. But that also cleans up the history, and makes clear what the result of the discussion is.

Reply:
In honestly, I'm a little worried about being overly specific here.

- On one hand, being clear in the bylaws does make expectations clear
- On the other hand, it requires changes to bylaws to update the process.

This language could be made a little clearer, such that the GH discussion path is an example: but even here, I have deviated a little bit, and we have to explain that.

When it moves over, I don't want to maintain conversation in BOTH the Discussion AND the PR (even contribution guide should be clearer about relationship between an issue or a discussion and a PR... )

In this case, I think closing the Discussion as resolved (clear to proceed to this as the impl) was the right thing to do, and the Discussion links here... 

How should we indicate or include that kind of flow?

Reply:
Firstly, I agree with the separation, I was asking to make sure I was understanding the intent correctly before commenting further.

I agree with the concern of changing the process and changing the by-laws for minor tweaks as it evolves and we decide on improvements. Is it sufficient to have the by-laws reference the contribution guide for the process of how the tool (GitHub) is to be used for making the decisions? Or do the by-laws need to have the details?

I know it then raises another problem of how do we handle process updates in the contribution guide. Which I don't have a good answer for right now

Reply:
Here is a thought, we can cross with lightweight ADR + Definition of Done (Y-statements): 

- https://www.ozimmer.ch/practices/2020/05/22/ADDefinitionOfDone.html
- https://medium.com/olzzio/y-statements-10eb07b5a177

As a way to articulate how to keep things moving forward and summarize actions and results? I can take a crack at that.

Reply:
Give it a crack. I'm not 100% sure I understand what you're thinking, but interested to see the result